tests/Unit/Command/ListFeatureCommandTest.php

```diff
@@ -118,9 +121,9 @@ final class ListFeatureCommandTest extends TestCase
 JSON,
                 'csv' => <<<CSV
 Manager,Name,Enabled,Description
-manager1,feature1,1,"Feature 1 description"
-manager1,feature2,,"Feature 2 description"
-manager2,feature3,1,"Feature 3 description"
+manager1,feature1,1,,"Feature 1 description"
```

Comment:
Too many commas for the column definitions, unless you want so many empty fields for the tests.